src/components/Signup/PodRegistrationForm.jsx

```diff
@@ -69,7 +70,7 @@ const PodRegistrationForm = ({ register, caseManagerName }) => {
         <p>You are not registering with anyone</p>
       )}
 
-      <Card variant="outlined" sx={cardStyle}>
+      <Card variant="none" sx={cardStyle}>
```

Comment:
Not sure if the "none" variant just the default card, but think we can remove this prop it is just the default value.

Reply:
I am not sure what i am removing.

Reply:
`variant="none"`. Was wondering if remove that is the same as the default MUI card. If it is, think wouldn't need the prop for `variant`. If we end up needing the prop after all, think we can keep `variant="none"`.

Reply:
Howzit, okay if i take that out, it goes back to the outlined box.



![Screenshot 2023-11-29 at 4 55 49 PM](https://github.com/codeforpdx/PASS/assets/68406221/f4646f6c-02db-4e80-a143-ceef921b10b0)

Reply:
Yeah, so i believe we need it. Are you good with me leaving it?

Reply:
Yeah, if it's needed, think we can keep it.